gax-java/.github/workflows/downstream.yaml

```diff
@@ -1,144 +0,0 @@
-on:
```

Comment:
We still want to keep this downstream check somewhere in the CI. I see that we already have downstream-native-image check in the main CI, we probably want to do the same for this downstream check.

Reply:
Yes, I'm bringing back the .github and .kokoro directories under all the modules. I thought the root .kokoro and .github directories have been configured completely and we can get rid of these individual ones.